### PR TITLE
nestjs sequelize integ tests

### PR DESCRIPTION
**Pull request by jhsinger-klotho**
nestjs sequelize integ tests and pulumi policy pack

### CODE REVIEW

File: ts-nestjs-sequelize/test/pulumi/index.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import * as aws from "@pulumi/aws";
+import { PolicyPack, validateResourceOfType, StackValidationPolicy } from "@pulumi/policy";
+
+const stackPolicy: StackValidationPolicy = {
+    name: "redis-test",
```
Comment (ewucc) [inline ts-nestjs-sequelize/test/pulumi/index.ts:5]: maybe typo? redis -> sequalize/rds

File: ts-nestjs-sequelize/test/pulumi/index.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        const lambdas = args.resources.filter(r => r.isType(aws.lambda.Function));
+        if (lambdas.length !== 1) {
+            reportViolation(`Expected one lambda function but found ${lambdas.length}`);
+            return;
+        }
+
+        const lambda = lambdas[0].asType(aws.lambda.Function)!;
+        if (lambda.memorySize !== 512) {
+            reportViolation(
+                `Expected rds Instance '${lambda.name}' allocated storage to be '512' but found '${lambda.memorySize}'`);
```
Comment (ewucc) [inline ts-nestjs-sequelize/test/pulumi/index.ts:18]: these messages say `rds Instance` but they're lambdas right

**Review by ewucc — CHANGES_REQUESTED** [review]

File: ts-nestjs-sequelize/test/pulumi/index.ts (hunk 1 above)
Comment (jhsinger-klotho) [reply ts-nestjs-sequelize/test/pulumi/index.ts]: this is just the lambda one so renamed. Will write utils/helpers tomorrow for all resources so we can reuse them.

File: ts-nestjs-sequelize/test/pulumi/index.ts (hunk 2 above)
Comment (jhsinger-klotho) [reply ts-nestjs-sequelize/test/pulumi/index.ts]: yeah fixed this

**Review by yuval-klotho — APPROVED** [review]

**Pull request merged**